static/js/vue-cdr-access/src/components/browseDisplay.vue

```diff
@@ -146,14 +147,15 @@
             },
 
             updateUrl() {
-                let params = this.urlParams({ types: 'Work' });
+                let type = this.is_collection ? 'Work' : 'Folder,File';
```

Comment:
Hmm, i'm not quite sure what this is doing. I see the same results whether i'm in a collection or folder, which is just works (which is correct), probably due to 
https://github.com/UNC-Libraries/Carolina-Digital-Repository/pull/805/files#diff-4d444f55adb40ebe5159fd69f6a9113bR159
But the url on a collection page has `type=Folder,File`. Could we remove this line and the insertion of the `types` param into the url on the next line, since retrieve data seems to win?

Reply:
Hmm, yeah it would have had to have been added in both places, since one updates the url and the other retrieves the actual data.